io_scene_niftools/modules/nif_export/property/object/__init__.py

```diff
@@ -240,15 +241,15 @@ def export_weapon_location(self, n_root, root_obj):
         # export weapon location
         if bpy.context.scene.niftools_scene.game in ('OBLIVION', 'FALLOUT_3', 'SKYRIM'):
             loc = root_obj.niftools.prn_location
-            if loc != "NONE":
+            if loc != "NONE" and (PRN_DICT[loc][bpy.context.scene.niftools_scene.game] is not None):
```

Comment:
bpy.context.scene.niftools_scene.game can be pulled out to a local variable.
`game = bpy.context.scene.niftools_scene.game`
used on line 242, 244, 248.